src/app/grid-column-moving/grid-column-moving.sample.html

```diff
@@ -36,6 +36,11 @@
                                                     [hasSummary]="false"
                                                     [dataType]="c.type">
                 </igx-column>
+                <igx-grid-footer>
+                    <div style='height:200px;'>
```

Comment:
Can this be applied directly to the `igx-grid-footer` element?

Reply:
@kdinev You can but the igx-grid-footer has display:inline by default so it would have no affect. You could set a different display and a height, for example:
` <igx-grid-footer style='height:200px;display:block;'>
      Test
   </igx-grid-footer> 
`